cobc/codegen.c

```diff
@@ -7531,6 +7582,10 @@ output_call (struct cb_call *p)
 		/* Dynamic link */
 		if (name_is_literal_or_prototype) {
 			s = get_program_id_str (p->name);
+			if(strncasecmp("Java.", s, 5) == 0) {
+				output_java_call(s + 5);
+				return;
+			} else {
```

Comment:
drop that else and the closing brace as you have a `return` already and don't need the additional scope (and create smaller diffs that way)